core/src/ast_builder/expr/function.rs

```diff
@@ -21,6 +22,13 @@ impl TryFrom<FunctionNode> for Expr {
                 .map(Function::Abs)
                 .map(Box::new)
                 .map(Expr::Function),
+            FunctionNode::IfNull(expr_node, then_node) => expr_node
+                .try_into()
+                .and_then(|expr| then_node
+                    .try_into()
+                    .map(|then| Function::IfNull { expr, then })
+                    .map(Box::new)
+                    .map(Expr::Function))
```

Comment:
It doesn't seem like rustfmt is applied.
Would you like to run `cargo fmt` ?

Reply:
@ever0de Yes, I totally forgot to do so.

I ran the command and pushed the commit. 

Thank you for the fast feedback.